WordPressAuthenticator.podspec

```
@@ -2,7 +2,7 @@
 
 Pod::Spec.new do |s|
   s.name          = 'WordPressAuthenticator'
-  s.version       = '2.3.0'
+  s.version       = '2.3.1-beta.1'
```

As Oguz @ Platform9 mentioned in the last code freeze https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/670#issue-1344624854, we want to follow the version scheme by bumping the middle digit for a new release. The last digit is only for hotfix. 
```suggestion
  s.version       = '2.4.0-beta.1'
```

---

Thanks for sharing this. 🙏 
Done in https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/672/commits/dcd1691969150c94fde7035b2d65dd9c6b782c03